listenbrainz/test_config.py

```diff
@@ -23,6 +23,7 @@
 # Redis
 REDIS_HOST = "redis"
 REDIS_PORT = 6379
+REDIS_NAMESPACE = "listenbrainz"
```

Comment:
Adding the config changes to the consul config template `consul_config.py.ctmpl` would be helpful too.